src/viewer.ts

```diff
@@ -879,14 +872,17 @@ class Viewer {
     }
 
     setTonemapping(tonemapping: string) {
-        const mapping = {
+        type mapping = {
+            [key: string]: number
+        };
+        const mappingLookup: mapping = {
```

Comment:
```suggestion
        const mappingLookup: Record<string, number> = {
```

I would remove the entire `mappingLookup` renaming to make it shorter again, but GitHub doesn't allow code suggestions for deleted lines.

Reply:
Ah cool suggestion, thanks, that's much better! 😄